_docs/05.2-further_reading.markdown

```diff
@@ -4,4 +4,54 @@ permalink: docs/further_reading.html
 toplevel: "Resources"
 ---
 
-Coming Soon!
+<br>
+#### Universal Javascript
+
+*  [Universal Javascript](https://medium.com/@mjackson/universal-javascript-4761051b7ae9#.vql04qjs4) - Defining things is hard: this articles outlines what Universal Javascript is and isn't. Naming things is hard: why developers are moving away from the term "Isomoprhic".
+
+*  [Why Everyone is Talking About Isomorphic / Universal JavaScript and Why it Matters](https://medium.com/capital-one-developers/why-everyone-is-talking-about-isomorphic-universal-javascript-and-why-it-matters-38c07c87905#.ut4ggn60w) - Short overview of the history of web development + why Universal Javascript is awesome.
+
+*  [Isomorphic JavaScript: The Future of Web Apps](http://nerds.airbnb.com/isomorphic-javascript-future-web-apps) - An overview of airbnb's case for Isomorphic (Universal) Javascript.
+
+#### React
+
+*  [React How To](https://github.com/petehunt/react-howto) - Info about the React ecosystem.
+
+*  [Thinking in React](https://facebook.github.io/react/docs/thinking-in-react.html) + [Design Principles](https://facebook.github.io/react/contributing/design-principles.html) - Read the React docs.
+
+*  [Building React.js at Enterprise Scale](https://medium.com/walmartlabs/building-react-js-at-enterprise-scale-17c17a36fd1f#.ewkzubo8i) - Why ReactJS + Walmart are a perfect match.
+
+#### Redux
+
+*  [A Cartoon Intro to Redux](https://code-cartoons.com/a-cartoon-intro-to-redux-3afb775501a6#.cj6szcbtd) - Essential Redux content + cartoons = Awesome!
+
+*  [Getting Started with Redux](https://egghead.io/courses/getting-started-with-redux) + [Building React Applications with Idiomatic Redux](https://egghead.io/courses/building-react-applications-with-idiomatic-redux) - This is a video series, but a great Redux introduction from the library's creator.
+
+#### Universal Rendering
+
+*  [ReactJS SSR Profiling + Caching](https://medium.com/walmartlabs/reactjs-ssr-profiling-and-caching-5d8e9e49240c#.yqnmwbp6t) - Outlines the [WalmartLabs](www.walmartlabs.com) SSR problem domain + tool created to solve this.
```

Comment:
better to use word `challenges` instead of `problem`

Server Side Rendering challenges + ways to solve this.